erpnext/public/css/pos.css

```diff
@@ -174,3 +174,6 @@ input[type=number]::-webkit-outer-spin-button {
 .rounded-total-value {
   font-size: 18px;
 }
+.quantity-total {
+  font-size: 18px;
```

Comment:
add in pos.less, not in pos.css

Reply:
Made suggested changes to sales_invoice.json, however when I add similar code to css.less I do not get similar styling and font for Total Qty. Refer screen shot below
![total_qty](https://user-images.githubusercontent.com/20460498/35976982-cdb77f0c-0d07-11e8-95ad-df5429576695.png)